packages/main/src/TextArea.hbs

```diff
@@ -12,7 +12,7 @@
 				{{this.text}}
 
 				{{#unless this.last}}
-					<br/>
+					\n
```

Comment:
I think `<br>` is the right one here, when you write HTML you never type `\n` for a new line.

Reply:
Done